server/api.go

```diff
@@ -49,15 +48,16 @@ type dexAPI struct {
 	serverConfig Config
 }
 
-type vehicleIDTokenClaims struct {
+type privilegeTokenClaims struct {
```

Comment:
Should we get rid of this struct? We could find a way to use it but right now I don't think you are. We are agnostic to the custom claims.